lib/scriptLoader.js

```diff
@@ -17,6 +17,9 @@
 
   function webWorkerImport(loader, load) {
     return new Promise(function(resolve, reject) {
+      if (load.metadata.integrity)
+        reject('cannot verify integrity of ' + load.address);
```

Comment:
Ideally, plugins could handle this as well. Right now, it only makes sense in plugin-css, however.